src/main/resources/static/stylesheet.css

```diff
@@ -2,5 +2,5 @@
 
 h1 {
 	text-align: center;
-	color: light-blue;
+	color: deepskyblue;

```

Comment:
I wasn't able to find light-blue as a qualified color name, so changed it to deepskyblue. Here's a list of color names if you're not interested in using the hex values. https://www.rapidtables.com/web/css/css-color.html

Reply:
Good catch.